frontend/src/components/team/SelectedTeam.jsx

```diff
@@ -37,16 +37,18 @@ const TeamCard = ({ data }) => {
     const LastStarted = last_started.slice(0, 16);
     const testStatusIcon = pickIcon(last_status);
     return (
-        <CardSection>
-            <header>
-                <H4>{name}</H4>
+        <CardSection aria-label={name} tabIndex="0">
```

Comment:
`tabindex="0"` enables the user to navigate to the element and therefore it receives focus, but nothing else happens when clicking the element/pressing the return key. Instead, the screen reader navigates to the selected build.

Reply:
I didn't realixe the focus style applies also when clicking with mouse. I'm open for better solution here.